dbt_project.yml

```diff
@@ -1,7 +1,7 @@
 name: 'dbt_utils'
 version: '0.1.0'
 
-require-dbt-version: [">=1.0.0", "<2.0.0"]
+require-dbt-version: [">=1.2.0-rc1", "<2.0.0"]
```

Comment:
I'm guessing this will drop the `-rc1` suffix and flip to just `>=1.2.0` prior to the final release of dbt_utils 0.9.0?

Reply:
Indeed! 
- dbt-utils 0.9.0-rc1 supports dbt-core 1.2.0-rc1. 
- dbt-utils 0.9.0 will support dbt-core 1.2.0